django_filters/rest_framework/backends.py

```diff
@@ -62,3 +62,10 @@ def to_html(self, request, queryset, view):
         }
         template = loader.get_template(self.template)
         return compat.template_render(template, context)
+
+    def get_fields(self, view):
```

Comment:
The original blame points to core-api/schema generation, but I'm not familiar with that so I don't actually understand how this method is being used. @tomchristie?

Side note - I saw @kevin-brown's comment [here](https://github.com/tomchristie/django-rest-framework/pull/4179#r69496876) and simplified.